pkg/framework/util.go

```diff
@@ -105,7 +109,111 @@ func NewUUID() string {
 
 // TestGetVersion test if we can get runtime name.
 func TestGetVersion(c internalapi.RuntimeService) {
-	version, err := c.Version(defaultAPIVersion)
+	_, err := c.Version(defaultAPIVersion)
 	ExpectNoError(err, "failed to get version: %v", err)
-	Logf("Get version runtime name " + version.RuntimeName)
+	Logf("Get version succeed")
+}
+
+// PodSandboxFound returns whether PodSandbox is found.
+func PodSandboxFound(podSandboxs []*runtimeapi.PodSandbox, podID string) bool {
+	if len(podSandboxs) == 1 && podSandboxs[0].Id == podID {
+		return true
+	}
+	return false
+}
+
+// buildPodSandboxMetadata builds PodSandboxMetadata.
+func buildPodSandboxMetadata(podSandboxName, uid, namespace string, attempt uint32) *runtimeapi.PodSandboxMetadata {
+	return &runtimeapi.PodSandboxMetadata{
+		Name:      podSandboxName,
+		Uid:       uid,
+		Namespace: namespace,
+		Attempt:   attempt,
+	}
+}
+
+// VerifyPodSandboxStatus verifies whether PodSandbox status for given podID matches.
+func VerifyPodSandboxStatus(c internalapi.RuntimeService, podID string, expectedStatus runtimeapi.PodSandboxState, statusName string) {
+	status := GetPodSandboxStatus(c, podID)
+	Expect(status.State).To(Equal(expectedStatus), "PodSandbox state should be "+statusName)
+}
+
+// RunPodSandbox runs a PodSandbox.
+func RunPodSandbox(c internalapi.RuntimeService, config *runtimeapi.PodSandboxConfig) string {
+	By("Run a PodSandbox.")
+	podID, err := c.RunPodSandbox(config)
+	ExpectNoError(err, "failed to create PodSandbox: %v", err)
+	Logf("Created PodSandbox %q\n", podID)
+	return podID
+}
+
+// RunDefaultPodSandbox runs a PodSandbox with default options.
+func RunDefaultPodSandbox(c internalapi.RuntimeService, prefix string) string {
+	podSandboxName := prefix + NewUUID()
+	config := &runtimeapi.PodSandboxConfig{
+		Metadata: buildPodSandboxMetadata(podSandboxName, defaultUID, defaultNamespace, defaultAttempt),
```

Comment:
Add random to sandbox ID/name/namespace, so the test cases could be easily paralleled later.